daffodil-cli/src/main/scala/org/apache/daffodil/Main.scala

```diff
@@ -319,40 +319,39 @@ class CLIConf(arguments: Array[String]) extends scallop.ScallopConf(arguments)
   shortSubcommandsHelp()
 
   // Global Options
-  val debug = opt[Option[String]](argName = "file", descr = "enable debugging. Optionally, read initial debugger commands from [file] if provided.")(optionalValueConverter[String](a => a))
-  val trace = opt[Boolean](descr = "run the debugger with verbose trace output")
-  val verbose = tally(descr = "increment verbosity level, one level for each -v")
-  val version = opt[Boolean](descr = "Show version of this program")
+  val debug = opt[Option[String]](argName = "file", descr = "Enable the interactive debugger. Optionally, read initial debugger commands from [file] if provided.")(optionalValueConverter[String](a => a))
+  val trace = opt[Boolean](descr = "Run this program with verbose trace output")
+  val verbose = tally(descr = "Increment verbosity level, one level for each -v")
+  val version = opt[Boolean](descr = "Show Daffodil's version")
 
   // Parse Subcommand Options
   object parse extends scallop.Subcommand("parse") {
-    banner("""|Usage: daffodil parse (-s <schema> [-r [{namespace}]<root>] [-p <path>] |
+    banner("""|Usage: daffodil parse (-s <schema> [-r [{<namespace>}]<root>] [-p <path>] |
```

Comment:
The square bracket, curly brace, and angle bracket is kind of hard to visiaully parse. Also, I think the standard is to use square brackets for optional things and angle brackets for required things, so I'm not sure the namespace is right. But the curly braces are also a bit confusing.

Maybe to simplify this and make it easier to visually parse, I wonder if we should just make this `-r <root>`. And then the help for the root option can describe the syntax (e.g. root element name, optionally preceeded by namespace surrounded in curly braces). Rarely do you actually need the namespace so it's probably not worth complicating the overview of options for this one rarely used feature.

If we do make this change, we should also do it any other time we allow a namespace to be specified. I think maybe the only other place is with the `-D` for variable names?